### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by ThomasGuldborg** [kirbydesign/designsystem#1468]
**Describe the bug**
The progress circle does not appear to display the remaining one percent

**To Reproduce**
Manipulate the value of the component to 99 % and then the progress circle appears to be filled out with 100 %

**Kirby version**
- [1.4.28]

**Expected behavior**
99 % should be visible with a remaining visible 1 %

**Screenshots**
<img width="137" alt="Skærmbillede 2021-04-15 kl  15 17 30" src="https://user-images.githubusercontent.com/77786415/114875650-cad25200-9dfd-11eb-9b14-d9fdeb010222.png">

**Please complete the following information:**
- OS: MacOS
- Browser Chrome 
- Browser version 89.0.4389.114

**Comment by bommariusser** [comment]
what is the usecase @ThomasGuldborg ? :)

**Comment by ThomasGuldborg** [comment]
For example when a customer has 1 % outstanding debt left, it should be visible, so that it does not look like that the loan is payed 100 % for :-)

**Comment by RasmusKjeldgaard** [comment]
@bommariusser @henrikvoetmand I think it is mainly a UX decision on what we should do in this case. 
I can verify that it is an issue. It happens due to the stroke width, and the rounded linecap. 

On a progress circle with small size, this problem occurs already for numbers 96 and up.

**Comment by bommariusser** [comment]
so this is probably also a usecase for 0-5% i guess @ThomasGuldborg ? i agree with you @RasmusKjeldgaard! 1-4% might be visualised as 5% and 95-99% might be visualised as 95%. Can you show 95%?

**Comment by RasmusKjeldgaard** [comment]
This is 95% for all sizes: 
![image](https://user-images.githubusercontent.com/42470636/116419987-db42ed80-a83d-11eb-9fc1-5b44a5bc7386.png)

0-5% will not be an issue, as it will still show the small individual steps here. It only becomes a problem when the linecaps meet at 96 and up.

**Comment by bommariusser** [comment]
so its probably something like:

95% on small
96-97% on medium 
97-98% on large

for it to be visually "seeable"? :D

**Comment by RasmusKjeldgaard** [comment]
Yes, but we could consider to align it so 95-99% is visualized as 95% across the board, to make it easier to understand and write in guidelines etc?
